.github/workflows/create-summary-pr.yml

```diff
@@ -0,0 +1,16 @@
+name: Create Automated Summary PR
```

Comment:
Shall we elaborate the topic?. Something like Create Automated Summary PR for the Load Test.

Reply:
Added in https://github.com/ballerina-platform/module-ballerina-grpc/pull/490/commits/f7e37371d95a24cdcb8e9d9d0c3f8e50704d90f7